### ISSUE TITLE
Save to Leave and Still Got Banned 

### ISSUE THREAD

**Issue by pkay01x** [ValveSoftware/Dota2-Gameplay#6930]
### Description

Opponent got disconnected so the match was considered not scored and it was safe to leave , I did leave the game but it gave me an abandon ? How ?

The Guy in the red marked (in the second screenshot uploaded ) is the one who disconnected and i got the punishments ?  

### Example Match ID (and possibly Timestamp)

Match ID : 7010479856 , 6:24PM (GMT+5:45)

### Screenshots

![dota1](https://user-images.githubusercontent.com/117821522/218097850-37e0e155-df45-484d-bace-b518107b0a31.png)
![dota21](https://user-images.githubusercontent.com/117821522/218097860-e0a8d00e-604d-4a07-82ac-d26d89f85b90.png)

**Comment by zeroedout** [comment]
Maybe a coincidence? Did your behavior score change after the match? It could have been your 15th game and maybe you got an overwatch conviction during the last 15?

**Comment by pkay01x** [comment]
> Maybe a coincidence? Did your behavior score change after the match? It could have been your 15th game and maybe you got an overwatch conviction during the last 15?

that's the funny part , actually right after this ban my behavior score updated , and it showed no signs of Abandon neither did it decrease but increased instead  
I had to screenshot it cause its so random. 
![summary](https://user-images.githubusercontent.com/117821522/218122331-f9b41fc0-a9dc-4068-9e4b-9ccc0d055f44.png)

**Comment by yckmm** [comment]
It's most likely a coincidence. If you abandon (and didn't abandon recently) the matchmaking ban is only 30 minutes.
Maybe you had a lot of comms reports, you can check here: https://steamcommunity.com/profiles/<YOUR_PROFILE_NUMBER>/gcpd/570/?category=Account&tab=MatchPlayerReportIncoming

**Comment by commentator24** [comment]
it's only updated on the next summary when you completed 15 games. e.g. that summary is from 15 games before, not the current one. perhaps you're at game 13...if you play 2 more it will show you had an abandon.

and I watched the replay a bit, you did receive an abandon.
![image](https://user-images.githubusercontent.com/109324155/218142712-e5d2f013-79cc-4fe2-ba00-52ced399eff6.png)

try complete 15? games until the next summary. see if I'm right whether you had an abandon. this is probably a bug to receive abandon after game is safe to leave, or because the coding is different for leave game button (it's only updated after game coordinator is updated or something)

and usually you dont have to click leave the game actually. it will update after few seconds dont know why you impatiently click leave game..? this also means i think it has a 5-15 sec delay because it's not instant. e.g. the button is only updated until game coordinator is updated i think. Always hover the ? before you leave game to see what punishment you'll get for clicking that leave game button.

for example i had a game where i suddenly dc to main menu, tried to reconnect but nothing happens. when i hover mouse to ? it only says if you leave you will be prevented to queue matchmaking for some time. i should've leave right away it has no other penalty like this.

I try reconnect again but it only sent me back to main menu again. So then I restart dota, Now it says if i leave i receive an abandon, receive a loss, teammate playing at disadvantage, prevented matchmaking... so i had to wait until it's updated... i think i waited like 5 min. so I do think it's not instant / bugged.

**Comment by pkay01x** [comment]
> it's only updated on the next summary when you completed 15 games. e.g. that summary is from 15 games before, not the current one. perhaps you're at game 13...if you play 2 more it will show you had an abandon.
> 
> and I watched the replay a bit, you did receive an abandon. ![image](https://user-images.githubusercontent.com/109324155/218142712-e5d2f013-79cc-4fe2-ba00-52ced399eff6.png)
> 
> try complete 15? games until the next summary. see if I'm right whether you had an abandon. this is probably a bug to receive abandon after game is safe to leave, or because the coding is different for leave game button (it's only updated after game coordinator is updated or something)
> 
> and usually you dont have to click leave the game actually. it will update after few seconds dont know why you impatiently click leave game..? this also means i think it has a 5-15 sec delay because it's not instant. e.g. the button is only updated until game coordinator is updated i think. Always hover the ? before you leave game to see what punishment you'll get for clicking that leave game button.
> 
> for example i had a game where i suddenly dc to main menu, tried to reconnect but nothing happens. when i hover mouse to ? it only says if you leave you will be prevented to queue matchmaking for some time. i should've leave right away it has no other penalty like this.
> 
> I try reconnect again but it only sent me back to main menu again. So then I restart dota, Now it says if i leave i receive an abandon, receive a loss, teammate playing at disadvantage, prevented matchmaking... so i had to wait until it's updated... i think i waited like 5 min. so I do think it's not instant / bugged.

Disconnecting is the way every time i leave Safe to Leave game , this is the first time i  faced the issues and got 5 LPs.
I waited for the Button to display green text " the game is safe to leave " also waited 4 seconds before actually clicking button which dota make you do . This might be a delay bug , still i got Punishments and LP cause i clicked button too fast  ?

**Comment by commentator24** [comment]
lol thats probably why u have low behaviour score then.

Try my way. Hover mouse to ? before you disconnect + click leave game button. 
If you don't see "you will receive an abandon" then it's safe to leave. if it still says you will receive an abandon, dont leave whatever happens, even when in game it says game is safe to leave or not safe to leave. Just wait patiently. you should only leave game after someone abandon when you dont get abandon as well.

as i've said before i think currently it's not updated instantly / synced. so yes you probably click leave too fast without reading the "?". i always follow the ? message and ignore display green text or not. if I still receive "abandon" i wont click leave.

Also displaying green text game is safe to leave only happens in normal matchmaking or lobbies i think it's a bit different in ranked. i havent seen it for years. Usually in ranked it will not turn green even after someone abandon. are you sure?

**Comment by pkay01x** [comment]
> lol thats probably why u have low behaviour score then.
> 
> Try my way. Hover mouse to ? before you disconnect + click leave game button. If you don't see "you will receive an abandon" then it's safe to leave. if it still says you will receive an abandon, dont leave whatever happens, even when in game it says game is safe to leave or not safe to leave. Just wait patiently. you should only leave game after someone abandon when you dont get abandon as well.
> 
> as i've said before i think currently it's not updated instantly / synced. so yes you probably click leave too fast without reading the "?". i always follow the ? message and ignore display green text or not. if I still receive "abandon" i wont click leave.
> 
> Also displaying green text game is safe to leave only happens in normal matchmaking or lobbies i think it's a bit different in ranked. i havent seen it for years. Usually in ranked it will not turn green even after someone abandon. are you sure?
 Yes im sure , i did click when it was green text * , i have had low behavior score cause of poor power cuts that happens here where i live so most of the time i don't i just don't leave and be extra safe when doing so , the only issue i faced is 
 > The guy disconnected and wasn't coming back so i went to main menu on dota to pass time  
 > The guy got abandon  
> I waited and made sure that it's safe to leave completely 
> Made sure it gave the message its safe to leave 
> Clicked Disconnect 
> Leave Game 
> Are you sure ? (4)
> You mean i got the message that "you will receive an abandon" even after safe to leave ? i think yes , maybe that's why i got abandon . 

Yes and im 100% sure , that it said this game is safe to leave now or something below the button .

**Comment by commentator24** [comment]
i'm talking about this 
![image](https://user-images.githubusercontent.com/109324155/218215483-43ac8dcd-a5b1-4e10-aa09-d2e6e7017f98.png)

**Comment by pkay01x** [comment]
> i'm talking about this ![image](https://user-images.githubusercontent.com/109324155/218215483-43ac8dcd-a5b1-4e10-aa09-d2e6e7017f98.png)

Yes there was a green text * safe to leave * 100%
